### ISSUE TITLE
dropout probelm

### ISSUE THREAD

**Issue by cgllv** [yxlijun/cifar-tensorflow#3]
I read your code carefully, and I find the dropout API have parameter training, the default value is false, does it influence training the network?
In addition, I think your code is beautifully written
